rpcd/playbooks/verify-maas.yml

```diff
@@ -23,7 +23,7 @@
         . {{ maas_venv_bin }}/activate
         {% endif %}
         {{ maas_rpc_scripts_dir }}/rpc-maas-tool.py verify-created \
-        --entity {{ inventory_hostname }} \
+        --entity {{ inventory_hostname }}{{ maas_fqdn_extension }} \
```

Comment:
The code in this repo is not used to create the MaaS entities. The entities are created as part of the server provisioning process prior to RPCO being deployed.

The templates do not know about the entity directly. The use of ansible_hostname or inventory_hostname in forming the label by the templates is simply to make it easier to tell which host a check belongs to when reading the API response data.

Checks/alarms get associated with an entity by the agent that uses the templates to create them.

[This](https://github.com/rcbops/rpc-openstack/blob/master/rpcd/playbooks/roles/rpc_maas/tasks/raxmon_agent_install.yml#L59-L134) section of code associates the entity with the agent and so with the host. The form of entity name used is "{{ inventory_hostname }}{{ maas_fqdn_extension }}". If this was not correct, the checks/alarms would not be created or would be associated with the incorrect entity.

My understanding is that it is only verification that fails, in which case using the same definition of entity name, as was used to create the checks/alarms, should be sufficient to solve this issue.